content/03-starting-oss-projects.md

```diff
@@ -143,11 +143,19 @@ You should have a policy for which of these way you use when. "Inbound=Outbound"
 
 ### Project governance
 
-See [issue #12](https://github.com/Open-Source-Compliance/outbound-oss/issues/12)
+An important factor for the success of an open source project is its governance. That comprises the rules, policies, conventions, and culture governing the collaboration. It determines factors such as how decisions are taken, who is in control, or who can join a project.
```

Comment:
Better: "... culture of the collaboration." instead of "... culture governing the collaboration".

Reply:
Changed it.